layouts/sessions/single.html

```diff
@@ -1,18 +1,67 @@
 {{ define "main" }}
     <div class="section-container">
-        <section class="section single-session-section">
-            <h1 class="single-session-section__heading">
-                <div class="single-session-section__metadata">
-                    {{- if page.Params.track }}
-                        <span class="single-session-section__hashtag">
-                            {{- partial "elements/hashtag.html" (dict "hashtag" page.Params.track) }}
-                        </span>
-                    {{- end }}
-                    {{- partial "elements/date-and-time.html" (dict "date" page.Params.startsAt ) }}
+        <section class="section ">
+            <div class="single-session-section-container">
```

Comment:
"container" is already an element, shouldn't it be "single-session-section__container"

Reply:
changed